multicast/exceptions.py

```
@@ -431,6 +433,82 @@
 		self.exit_code = 143  # Use SIGTERM exit code for graceful shutdown
 
 
+def validate_exit_code(code) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `code`

(ANN001)